cmd/templates/hook.tmpl

```diff
@@ -14,24 +14,24 @@ call_lefthook()
 {
   if lefthook -h >/dev/null 2>&1
   then
-    eval $1
-  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/lefthook"
+    eval lefthook $1
```

Comment:
Should we also add `{{.Extension}}` here (and in `if` above)? I think yes, because if someone will install Lefthook system-wide in Windows, it can be called only via `lefthook.exe` then.
```suggestion
    eval lefthook{{.Extension}} $1
```